src/components/landing/FeatureBlocks/AlternatingFeatureBlock.js

```diff
@@ -27,6 +28,15 @@ const FeatureBlock = ({
       })}
     >
       <Title el="h3" className="xl:text-2xl xl:tracking-tight text-primary-600">
+        {comingSoon && (
+          <>
+            <span className="inline-block border border-blueroadie text-blueroadie mb-2 text-sm p-2 rounded-lg">
+              Coming soon
```

Comment:
What's the objective of adding "Coming soon" here? I think it kills the entire page to put it here, I would immediately close the page tbh